tests/environment/docker-compose-pipeline-all-amps.yml

```diff
@@ -0,0 +1,201 @@
+                -Dsfs.url=http://shared-file-store:8099/
+                -DlocalTransform.core-aio.url=http://transform-core-aio:8090/
+                -Dalfresco-pdf-renderer.url=http://transform-core-aio:8090/
+                -Djodconverter.url=http://transform-core-aio:8090/
+                -Dimg.url=http://transform-core-aio:8090/
+                -Dtika.url=http://transform-core-aio:8090/
+                -Dtransform.misc.url=http://transform-core-aio:8090/
+                -Dcsrf.filter.enabled=false
+                -Ddsync.service.uris=http://localhost:9090/alfresco
+                -Ds3.accessKey=${AWS_ACCESS_KEY_ID}
+                -Ds3.secretKey=${AWS_SECRET_ACCESS_KEY}
+                -Ds3.bucketName=pipeline-amps-all
+                -Ds3.autoLowerCaseBucketName=true
+                -Ds3.bucketLocation=us-east-1
+                -Ds3.abortIncompleteMultipartUploadDays=1
+                -Dsaml.sp.isEnabled=false
+                -Xms1500m -Xmx1500m
+                "
+        ports:
+          - 10001:10001
+
+    transform-router:
+        mem_limit: 512m
+        image: quay.io/alfresco/alfresco-transform-router:1.2.0
+        environment:
+          JAVA_OPTS: " -Xms256m -Xmx512m"
+          ACTIVEMQ_URL: "nio://activemq:61616"
+
+          CORE_AIO_URL : "http://transform-core-aio:8090"
+
+          FILE_STORE_URL: "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file"
+        ports:
+            - 8095:8095
+        links:
+          - activemq
+
+    transform-core-aio:
+        image: alfresco/alfresco-transform-core-aio:2.2.1
+        mem_limit: 1536m
+        environment:
+            JAVA_OPTS: " -Xms256m -Xmx1536m"
+            ACTIVEMQ_URL: "nio://activemq:61616"
+            FILE_STORE_URL: "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file"
+        ports:
+            - 8090:8090
+        links:
+        - activemq
+
+    shared-file-store:
+        image: alfresco/alfresco-shared-file-store:0.7.0
+        mem_limit: 512m
+        environment:
+            JAVA_OPTS: " -Xms256m -Xmx512m"
+            scheduler.content.age.millis: 86400000
+            scheduler.cleanup.interval: 86400000
+        ports:
+            - 8099:8099
+        volumes:
+            - shared-file-store-volume:/tmp/Alfresco/sfs
+
+    share:
+        image: alfresco/alfresco-pipeline-all-amps-share:${TRAVIS_BUILD_NUMBER}-${PROJECT_VERSION}
+        mem_limit: 1g
+        environment:
+            REPO_HOST: "alfresco"
+            REPO_PORT: "8080"
+            JAVA_OPTS: "
+                -Xms500m
+                -Xmx500m
+                -Dalfresco.host=localhost
+                -Dalfresco.port=8080
+                -Dalfresco.context=alfresco
+                -Dalfresco.protocol=http
+                "
+            TRAVIS_BUILD_NUMBER: "${TRAVIS_BUILD_NUMBER}"
+
+    postgres:
+        image: postgres:11.4
```

Comment:
I believe we're moving to Postgres 11.7 for the latest releases so please update this reference. As Alan says this file should be as close as possible to https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/docker-compose.yml

Reply:
Done